go-controller/pkg/ovn/controller/services/loadbalancer_test.go

```diff
@@ -40,8 +40,8 @@ func TestEnsureLBs(t *testing.T) {
 			Protocol:    "TCP",
 			Rules: []LBRule{
 				{
-					Source:  Addr{"1.2.3.4", 80},
-					Targets: []Addr{{"169.254.169.2", 8080}},
+					Source:  Addr{IP: "1.2.3.4", Port: 80},
+					Targets: []Addr{{IP: "169.254.169.2", Port: 8080}},
```

Comment:
super NIT: this was not needed, but I like having it explicitly mentioned like that too. :)

Reply:
Ecactly... and I had to add the field names everywhere :(